### PR TITLE
KATI-14_Doctor-Profile-View-Feature

### PR DESCRIPTION
**Pull request by vtnlinh95**

### CODE REVIEW

File: app/src/main/java/com/kms/cura/view/adapter/WorkingHourExpandableAdapter.java [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+        }
+
+        TextView dayOftheWeek = (TextView) convertView.findViewById(R.id.txtDayofTheWeek);
+        LinearLayout workingTimeLayout = (LinearLayout) convertView.findViewById(R.id.layoutWorkingTime);
+        if(workingTimeLayout.getChildCount() > 0){
+            workingTimeLayout.removeAllViews();
+        }
+        dayOftheWeek.setText(DayOfTheWeek.getDayOfTheWeek(childPosition).toString().substring(0,3));
+        Set<String> facilitiesName = workingHours.keySet();
+        for(String facilityName : facilitiesName){
```
Comment (hieumai) [inline app/src/main/java/com/kms/cura/view/adapter/WorkingHourExpandableAdapter.java]: re-format this file

**Comment by hieumai** [comment]
OK from me. @huuduy216  and @toannguyen71195  please start verifying on this

**Comment by huuduy216** [comment]
@vtnlinh95 
- A small section of working hour( Sunday) is lost
- When login as a doctor, the app doesn't automatically change to profile view (user need to click on profile from navigation drawer)
- No profile view is shown when click on doctor search result ( as a patient user yet)

**Comment by vtnlinh95** [comment]
@huuduy216  : 
untitled
I don't see the Sunday section lost

**Pull request merged**